core/Lexer.ts

```diff
@@ -50,6 +50,10 @@ export default class Lexer {
       const matched = line.match(regex);
 
       if (matched && matched[0]) {
+        if (!line.startsWith(matched[0])) {
+          throw new Error(`No value on the position ${this.pos} specified`);
```

Comment:
Grammar mistake. The same is for other usages of 'the' after 'on'.
```suggestion
          throw new Error(`No value on position ${this.pos} specified`);
```